packages/docs/site/docs/01-start-here/03-repos-overview.md

```diff
@@ -0,0 +1,79 @@
+
+Searching for issues is a great way to find a fix or workaround, or to see if a feature request has already been made. If you find an issue that matches your problem, you can add a comment to the issue to provide more information or to show that you are also experiencing the same problem.
+
+Explore the WordPress Playground [repository](https://github.com/WordPress/wordpress-playground/)
+
+---
+
+## [Playground Tools]((https://github.com/WordPress/playground-tools/))
+This repository contains a collection of tools and utilities that are used to manage and maintain the WordPress Playground project. The tools include scripts for building and deploying the project, as well as utilities for managing the documentation and other project assets.
+
+There are also advanced administration features, integrations such as the Playground Plugin, Blueprint Builder, and Interactive Code Block that live in the Playground Tools repository.
+
+This repository is also the best place to search for issues related to `wp-now` or the `VS Code Extension` for WordPress Playground. If the request you are looking for is not specific to the online version of the Playground, it may be in the Playground Tools repository.
+
+#### Tools that exist in the Playground Tools repository:
+- [Blueprint Builder](https://playground.wordpress.net/builder/builder.html) - a tool to create a view Blueprints side-by-side
+- [Playground Block](https://wordpress.org/plugins/interactive-code-block/) - embed Playground via a block on your WordPress site
+- [Interactive Code Block](https://wordpress.org/plugins/interactive-code-block/) - an interactive code editor to demonstrate and teach your readers how WordPress plugins are built.
+- [Playground Plugin](https://wordpress.org/plugins/playground/) - Copy your site or test plugins easily
+- [VS Code Extension](https://marketplace.visualstudio.com/items?itemName=WordPressPlayground.wordpress-playground) - Easily spin up WordPress instances in VS Code
+- [`wp-now` ](https://www.npmjs.com/package/%40wp-now/wp-now)- a fully featured, extensible environment for WordPress development
+- [Pull Request Previewer](https://playground.wordpress.net/wordpress.html) - a tool to preview pull requests in the Playground
+- etc.
+
+The list of Playground Tools continues to grow by the day, and we cannot wait to see what exciting tools are added next. Want to contribute? Swing on by the Playground Tools [repository](https://github.com/WordPress/playground-tools/) and see what you can do to help.
+
+---
+
+## [Blueprints Library](https://github.com/WordPress/blueprints-library)
+The Blueprints Library is a collection of pre-configured WordPress setups that can be used to quickly create a new WordPress site in the Playground. Each blueprint contains a set of instructions for setting up a specific type of WordPress site, along with the necessary configuration files.
```

Comment:
Ah this one is a library in a "PHP library" sense, perhaps we could come with a better name. The Blueprints gallery/community space is at https://github.com/adamziel/blueprints and will be moved over under the WordPress org soon

Reply:
I meant a library as in https://en.wikipedia.org/wiki/Library_(computing), not as a metaphor for Blueprints

Reply:
Vastly improved